.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+- [ ] Refer to the issue that supports this Pull Request.
```

Comment:
Same here. Maybe add fixed data sections to indicate issue link, pull request to docs...

Reply:
Same as above, I would see how it goes. It can be iterated, if people is not organized we can change it

Reply:
ok